File: Tutorials/WinForms/DevExpress.MVVM/CS/MainView.cs
using DevExpress.XtraBars.Ribbon;
using WinFormsApplication.ViewModels;

namespace WinFormsApplication {
    public partial class MainView : RibbonForm {
        public MainView() {
            InitializeComponent();
            if(!mvvmContext1.IsDesignMode)
                InitializeBindings();
        }
        void InitializeBindings() {
            var fluentApi = mvvmContext1.OfType<MainViewModel>();
            fluentApi.WithEvent(this, nameof(Load))
                .EventToCommand(x => x.ShowCustomers());
            //
            fluentApi.BindCommand(customersBarButtonItem, x => x.ShowCustomers());
            fluentApi.BindCommand(customersAccordionControlElement, x => x.ShowCustomers());
            fluentApi.BindCommand(ordersBarButtonItem, x => x.ShowOrders());
            fluentApi.BindCommand(ordersAccordionControlElement, x => x.ShowOrders());
        }
    }
}

File: Tutorials/WinForms/DevExpress.MVVM/VB/MainView.vb
Imports DevExpress.XtraBars.Ribbon
Imports WinFormsApplication.ViewModels

Namespace WinFormsApplication
	Partial Public Class MainView
		Inherits RibbonForm

		Public Sub New()
			InitializeComponent()
			If Not mvvmContext1.IsDesignMode Then
				InitializeBindings()
			End If
		End Sub
		Private Sub InitializeBindings()
			Dim fluentApi = mvvmContext1.OfType(Of MainViewModel)()
			fluentApi.WithEvent(Me, NameOf(Load)).EventToCommand(Sub(x) x.ShowCustomers())
			'
			fluentApi.BindCommand(customersBarButtonItem, Sub(x) x.ShowCustomers())
			fluentApi.BindCommand(customersAccordionControlElement, Sub(x) x.ShowCustomers())
			fluentApi.BindCommand(ordersBarButtonItem, Sub(x) x.ShowOrders())
			fluentApi.BindCommand(ordersAccordionControlElement, Sub(x) x.ShowOrders())
		End Sub
	End Class
End Namespace
